store/src/java/com/zimbra/cs/service/account/GetPrefs.java

```diff
@@ -98,6 +99,8 @@ public static void doPrefs(Account acct, Element prefs, Map<String, Object> attr
                 prefs.addKeyValuePair(key, (String) value, AccountConstants.E_PREF, AccountConstants.A_NAME);
             }
         }
-    }   
+        prefs.addKeyValuePair(LC.zimbraPref15MinutesCalenderScaling.key(), String.valueOf(LC.zimbraPref15MinutesCalenderScaling.booleanValue()),
```

Comment:
can we change preference key to zimbraPrefCalenderScaling and value to either "15" if LC value is true and "30" if LC value is false (don't change LC key/value), this way same preference can be used in 9.1 as ldap attribute, where it will be an enum containing all possible values like 5,10,15,30 etc.

Reply:
Done